packages/tools/src/types/SegmentationStateTypes.ts

```diff
@@ -77,6 +77,10 @@ export type Segmentation = {
    * only labelmap representation is supported)
    */
   representationData: SegmentationRepresentationData;
+  /**
+   * volume id of the segmentation reference series
+   */
+  referencedVolumeId?: string;
```

Comment:
a Segmenation object should not be genrarlized to be a labelmap. we already have SegmentationRepresentationData in which it can be labelmap or contour. And in the LabelmapSegmentationData we already have the referencedVolumeId

Reply:
There is two points here: the referenced series data will always be an dicom image data, where we can get the dimensions and other properties, if any not descriped in representationalData. So, we can call it referencedVolumeId, And it should be valid independent of the segmentation representation, so it would be best if it goes in segmentation level, not in representational level. And although the representationData has the field, currently the value has not been set.